src/get.sh

```diff
@@ -1,7 +1,53 @@
 #!/bin/sh
 
-# latest GA release
-KEPTN_VERSION=${KEPTN_VERSION:-"0.7.3"}
+# Define handy functions
+get_latest_version(){
+   curl --silent "https://api.github.com/repos/keptn/keptn/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/' 
```

Comment:
This command is not working on MacOS, see https://travis-ci.com/github/keptn/get.keptn.sh/jobs/461956400

I thinkt he problematic part is the `sed -E` command. Take a look at the script I copied from istio here https://github.com/keptn/get.keptn.sh/issues/24, they are not using sed, but a combination of grep and tail.

Reply:
I verified on Mac OS 10.14.6 and it is working. 
```
juergen.etzlstorfer$ curl --silent "https://api.github.com/repos/keptn/keptn/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/'
0.7.3
```

Which OS version is used in travis?

Reply:
Thanks Jüergen. I think travis-ci has a weird version of awk available. Though I'm not sure if people might have this version, or if everyone has the correct version ...

Anyway, Travis-CI osx is running 
```
Operating System Details
ProductName:	Mac OS X
ProductVersion:	10.13.6
BuildVersion:	17G65
```